src/test/java/project/web/GasStationControllerTest.java

```diff
@@ -46,11 +46,10 @@ void getGasStationByNameTestWithInvalidInput() {
         String stationName = "Non-existent Station";
         when(gasStationService.getStationsByName(stationName)).thenReturn(new ArrayList<>());
 
-        //WHEN
-        ResponseEntity<List<GasStationDTO>> response = gasStationController.getGasStationByName(stationName);
-
-        //THEN
-        assertEquals(HttpStatus.NOT_FOUND, response.getStatusCode());
+        // WHEN THEN
+        assertThat(gasStationController.getGasStationByName(stationName))
```

Comment:
i would keep the WHEN separated from the THEN

Reply:
It’s fixed and split.